js/deer-utils.js

```diff
@@ -167,11 +167,31 @@ export default {
      * Execute query for any annotations in RERUM which target the
      * id passed in. Promise resolves to an array of annotations.
      * @param {String} id URI for the targeted entity
+     * @param [String] targetStyle other formats of resource targeting.  May be null
      */
-    findByTargetId: async function (id) {
+    findByTargetId: async function (id, targetStyle) {
         let everything = Object.keys(localStorage).map(k => JSON.parse(localStorage.getItem(k)))
-        let obj = {
-            target: id
+        //Backwards compatible without the new parameter
+        if(targetStyle === undefined){
+            targetStyle = []
+        }
+        //Users pass the craziest things...
+        if (!Array.isArray(targetStyle)) {
+            if(typeof targetStyle === "string"){
```

Comment:
You already discard non-strings later on, so you can just fire the next line regardless.